scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+export SKIP_USER_LOGIN_TESTS="true"
+export GINKGO_TEST_ARGS="--noColor"
+
+shout "Getting oc binary"
+if [[ $BASE_OS == "linux"  ]]; then
```

Comment:
Where/How is this variable being set to `linux` or anything else?

Reply:
This value is injected on jenkins and is expected to be set already

Reply:
Ack, thanks.